lib/src/views/misc/interactive_grid_tile.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:rescado/src/views/buttons/floating_button.dart';
+
+class InteractiveGridTile extends StatefulWidget {
+  final String image;
+  final FloatingButton floatingButton;
+
+  const InteractiveGridTile({Key? key, required this.image, required this.floatingButton}) : super(key: key);
```

Comment:
Separate lines for >1 arguments please.

Reply:
Also this should not be under views/misc but views/containers. I would also drop "interactive" from the name and go with just `grid_tile` or even `grid_item` to imply the link with `list_item`.